app/components/EllipsizedContent/index.tsx

```diff
@@ -72,6 +71,8 @@ function EllipsizedContent(props: Props) {
         }
     }, [children, maxCharacters, isEllipsized]);
 
+    const handleShowEllipsized = useCallback(() => setIsEllipsized(!isEllipsized), [isEllipsized]);
```

Comment:
Let's just use function

```suggestion
    const handleShowEllipsized = useCallback(() => setIsEllipsized((oldVal) => !oldVal), []);
```